HTML/styles.scss

```diff
@@ -0,0 +1,53 @@
+.level1 {
+    padding: 10px;
+    background-color: #f0f0f0;
+
+    .level2 {
+        padding: 10px;
+        background-color: #d0d0d0;
+
+        .level3 {
+            padding: 10p;
```

Comment:
<div>

<strong>Bito Code Review Agent Run</strong> <a href=https://github.com/Amruta101998/DS-and-Algorithm/pull/17#issuecomment-2188976663>#automa</a><span> - 06/25/2024, 01:40 pm</span>
<br>

<div><small>🔴</small> High importance</div>

<div id="suggestion">
<div id="issue">
<b>Issue:</b> Typographical error in the padding property for '.level3'. The unit 'p' is incorrect and should be 'px'.
</div>
<div id="fix">
<b>Fix:</b> Correct the unit for the padding property to 'px'.
</div>
<details>
<summary>
<b>Code suggestion</b>
</summary>
<div id="code">

```
 @@ -10,7 +10,7 @@
      padding: 10px;
  }
```
</div>
</details>
</div>
<br>


</div>